config/api.yml

```diff
@@ -477,10 +477,17 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
```

Comment:
The only reason I have the  `delete` here is to avoid the below error.
That's the only purpose it serves.

```ruby
Failures:

  1) Cloud Volumes API can delete cloud volumes through POST
     Failure/Error:
       Api::ApiConfig.collections[type][selection][method]
         .detect { |spec| spec[:name] == action.to_s }[:identifier]
     
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./spec/support/api_helper.rb:113:in `action_identifier'
     # ./spec/support/api_helper.rb:117:in `collection_action_identifier'
     # ./spec/requests/api/cloud_volumes_spec.rb:51:in `block (2 levels) in <top (required)>'
```

I have also tried using 
```ruby
api_basic_authorize action_identifier(:cloud_volumes, :delete, :resource_actions, :post)
``` 
but that requires the `delete` in `collection_actions` as well.